index.html

```diff
@@ -6,6 +6,23 @@
     <title>Nep huiswerkopdracht</title>
 </head>
 <body>
+<h>Titel van pagina</h>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam corporis libero sapiente. Aut laudantium placeat
```

Comment:
Ik zou de Lorem ipsum niet gebruiken en gewoon telkens een eigen tekst schrijven. Dat is véél gemakkelijker. 
Graag aanpassen ;)